dynacred/Makefile

```diff
@@ -4,10 +4,19 @@ deps:
 none:
 	@echo "Not breaking if you don't tell me what to do"
 
-src:
+cothority-pull:
 	make -C .. cothority-pull
-	ln -s ../../../cothority/external/js/cothority/src src/lib/cothority
-	ln -s ../../../cothority/external/js/kyber/src src/lib/kyber
+
+src: cothority-pull src/lib/cothority src/lib/kyber
```

Comment:
Ha, a good `.PHONY` use: as you do `make src` and the directory already exists, it won't be executed. In this specific case, it will not occure because it depends on `cothority-pull` which is a file never created by the `cothority-pull` rule (it should also be `.PHONY`).

btw, one nice way to see what make is considering/executing/checking is to use `make --debug`, so, in our case
```
[...]
Reading makefiles...
Updating makefiles....
Updating goal targets....
   File 'cothority-pull' does not exist.
  Must remake target 'cothority-pull'.
make -C .. cothority-pull
[...]
cd cothority && git pull
Already up to date.
Current branch master is up to date.
[...]
  Successfully remade target file 'cothority-pull'.
 Prerequisite 'cothority-pull' of target 'src' does not exist.
 Prerequisite 'src/lib/cothority' is newer than target 'src'.
 Prerequisite 'src/lib/kyber' is newer than target 'src'.
Must remake target 'src'.
Successfully remade target file 'src'.
```